frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pages/citizen/Create/OtherDetailsSection.js

```
@@ -0,0 +1,137 @@
+  styles,
+  formErrors
+}) => {
+  const stateId = Digit.ULBService.getStateId();
+  const { data: Menu = {}, isLoading } = Digit.Hooks.pt.usePropertyMDMS(stateId, "PropertyTax", "PTPropertyType") || {};
+  const [propertyTypeOptions, setPropertyTypeOptions] = useState([]);
+
+  useEffect(() => {
+    if (Menu?.PropertyTax?.PropertyType?.length) {
+      const options = Menu.PropertyTax.PropertyType.map((item) => ({
+        code: item.code,
+        name: t("COMMON_PROPTYPE_" + item.code.replace(/\./g, "_")),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)